File: src/main/java/com/kharcha/kharcha/Entities/Expenses.java
package com.kharcha.kharcha.Entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;


import java.util.Date;

@Entity
@Data

public class Expenses {

    @Id
    @GeneratedValue(
            strategy = GenerationType.IDENTITY
    )
    private Integer expenseId;
    private Integer amount;
    private Date date;
    private String title;

}

File: frontend/src/components/Expenses/Expenses.js
import React from 'react';
import Card from '../UI/Card';
import ExpenseItem from './ExpenseItem';
import './Expenses.css';

const Expenses=(props)=>{
    return  <Card class="expenses">
                {
                        props.item.map(
                                expense=>(
                                        <ExpenseItem
                                        key={expense.expenseId}
                                        date={expense.date}
                                        title={expense.title}
                                        amount={expense.amount}/>
                        )
                        )
                }
        </Card>

}

export default Expenses;